src/Utils/__tests__/useCollectorSignals.jest.tsx

```diff
@@ -135,16 +115,9 @@ describe("useCollectorSignals", () => {
       }),
     })
 
-    const { result } = renderHook(
-      () =>
-        useCollectorSignals({
-          artwork: hookProps.data?.artwork,
-          me: hookProps.data?.me,
-        }),
-      { wrapper }
-    )
+    const result = getResult()
 
-    expect(result.current).toEqual({
+    expect(result).toEqual({
```

Comment:
Nonblocking, but we could organize the test file a bit better so we don't need to alias fields and keep stuff out of the describe scope:

```
describe('useCollectorSignals', () => {
  const mockUseFeatureFlag = useFeatureFlag as jest.Mock

  const TestApp = props => {
    const result = useCollectorSignals(props)
    return <div data-testid="result">{JSON.stringify(result)}</div>
  }

  const getResult = () => JSON.parse(screen.getByTestId("result").textContent!)
  
  describe('multiple artworks', () => {
    const { renderWithRelay } = setupTestWrapperTL({
      Component: TestApp,
      query: graphql`
        query useCollectorSignalsArtworksConnectionTestQuery {
          artworksConnection(first: 3) {
            __typename
            ...useCollectorSignals_artworksConnection
          }
          me {
            ...useCollectorSignals_me
          }
        }
      `,
    })
    
    it(...) 
  })
  
  describe('single artwork', () => {
    const { renderWithRelay } = setupTestWrapperTL(...)
    
    it('does other stuff') 
  })
})